core/app/models/spree/return_item.rb

```diff
@@ -36,6 +36,7 @@ class ReturnItem < ActiveRecord::Base
     belongs_to :reimbursement, inverse_of: :return_items
     belongs_to :preferred_reimbursement_type, class_name: 'Spree::ReimbursementType'
     belongs_to :override_reimbursement_type, class_name: 'Spree::ReimbursementType'
+    belongs_to :return_authorization_reason, class_name: 'Spree::ReturnAuthorizationReason', foreign_key: :return_authorization_reason_id
```

Comment:
just curious, why bring 'authorization' into the name here? it could also be during an actual return, not necessarily only during a return authorization.